lib/AvatarPresence/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
+import Tooltip from '../Tooltip';
+
+const tooltipOnHover = (data, isAssigned) => (
+  <Tooltip className="avatar__tooltip" id={data.email}>
+    <p className="avatar__tooltip-name">{data.name}</p>
+    <p className="avatar__tooltip-email">{data.email}</p>
+    { isAssigned &&
+      <span className="avatar__pillbox">Assigned</span>
+    }
+  </Tooltip>
+);
+
+const AvatarPresence = (props) => {
```

Comment:
What's your thinking around the name `AvatarPresence`?

I noticed that in the stories area we use this alongside a `AvatarPresenceGroup` component but the file is called `AvatarGroup`. Hitting it with a simple stick I thought of names such as;

- Avatar (with added initials & custom colour support)
- AvatarGroup (handles collapsing etc)
- AvatarWithTooltip (provides a tooltip with name/email + additional data if required e.g. assigned)

In the UI scope where does "presence" come into play? Once I understand that I think I'll get your thinking behind the name a little more 👍 

-------------------

A good example of the ComponentWithTooltip approach is the `ButtonWithTooltip` component.